tests/contrib/mysql/test_mysql.py

```diff
@@ -58,8 +58,8 @@ def test_simple_query(self):
             },
         )
 
-    def test_simple_query_fetchll(self):
-        with self.override_config("mysql", dict(trace_fetch_methods=True)):
+    def test_simple_query_fetchall(self):
+        with self.override_config("dbapi2", dict(trace_fetch_methods=True)):
```

Comment:
why this change, seems unrelated? this shouldn't be changing

Reply:
`trace_fetch_method` has been removed from `dbapi2` and integration-specific configuration should be used. Is this causing issues with tests?

Reply:
reverted it, but kept function name change